File: lib_common-utils/src/main/java/com/common/utils/audio/RecordCallback.java
package com.common.utils.audio;

public interface RecordCallback {
    /**
     * 数据回调
     *
     * @param bytes 数据
     * @param len   数据有效长度，-1时表示数据结束
     */
    void onRecord(byte[] bytes, int len);
}

File: lib_common-utils/src/main/java/com/common/utils/audio/RecordCallback.kt
package com.common.utils.audio

fun interface RecordCallback {
    /**
     * 数据回调
     *
     * @param bytes 数据
     * @param len   数据有效长度，-1 时表示数据结束
     */
    fun onRecord(bytes: ByteArray, len: Int)
}